### DISCUSSION TITLE
any way to make the next video of a playlist stay fullscreen when the video you are watching ends?

### DISCUSSION THREAD

**Discussion by M-Hamano16** [FreeTubeApp/FreeTube#5543] [General]
I like to make playlists and watch them when laying down.  I put the player on full screen mode, but when one video ends it goes smaller after changing to the next video.  Is there a way to keep things completely on full screen mode?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @M-Hamano16, no unfortunatly this isnt possible at the moment. We have open issues for this though, #2295 #5427
